.rspec

```diff
@@ -1,2 +1,3 @@
 --format documentation
 --color
+--order random
```

Comment:
oh wait I am looking at the wrong repo! Well, can we add this via `spec_helper` instead of `.rspec`? That's probably where I'd expect to see  it